insalan/tournament/views/tournament.py

```diff
@@ -354,8 +354,8 @@ def get(self, request, primary_key: int):
 
                 swiss["teams"] = SwissSeeding.objects.filter(swiss=swiss["id"]).values_list("team", flat=True)
 
-            tourney_serialized["seatings"] = serializers.SeatingSerializer(
-                Seating.objects.filter(tournament=tourney), context={"request": request}, many=True
+            tourney_serialized["seatslots"] = serializers.SeatSlotSerializer(
```

Comment:
You also need to add the list of seats for each slots in order for the front to compute them.
You can take example on the team list in the same endpoint to dereference them.